core/src/blocktree.rs

```diff
@@ -408,6 +408,7 @@ impl Blocktree {
                                 ),
                             )
                             .to_owned(),
+                        log::Level::Error,
```

Comment:
Can this use `solana_metrics::datapoint()` instead?

Reply:
Yes, it can/should be changed to datapoint!